README.md

```diff
@@ -235,6 +253,22 @@ Type comprising every "plain" Object value. Specifically, values created via:
   - the `new` operator in conjunction with `Object` or a custom
     constructor function.
 
+#### `PositiveInteger`
+
+```haskell
+$.PositiveInteger :: Type
+```
+
+Type comprising every [`Integer`](#integer) value greater than zero.
+
+#### `NegativeInteger`
+
+```haskell
+$.NegativeInteger :: Type
+```
+
+Type comprising every [`Integer`](#integer) value less than zero.
```

Comment:
This is a clear, succinct description. :)

I do wonder, though, whether we should obey strict alphabetical ordering. If we're to group *some* related types it seems to me we should group *all* related types. Grouping has the advantage of making the types section more cohesive upon first reading, but it makes it harder to find a section by name. It also forces us to decide to which group a type belongs each time we add one. Strict alphabetical ordering seems right for now. If we decide to group the types we should discuss this as a separate change.